src/index.mjs

```diff
@@ -113,6 +114,20 @@ export class Circuit extends HeadlessCircuit {
     displayOn(elem) {
         return this._makePaper(elem, this._graph);
     }
+    zoomIn(paper) {
+        this.currentZoomIndex++;
```

Comment:
Why is this defined for the whole circuit? Shouldn't this value be per paper? There is room for confusion here - you are getting the paper from a parameter, but are changing some global value, possibly unrelated to this paper.

Reply:
I think that the zoomIn/zoomOut methods are not really needed, and just the scaling method is enough. This would also force the API user to keep the index locally.

Reply:
In the recent commits I removed the zoomIn and zoomOut methods. I moved the responsibility of remembering the scale to the caller.